applications/GeoMechanicsApplication/custom_elements/interface_stress_state.cpp

```diff
@@ -0,0 +1,60 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "interface_stress_state.h"
+
+namespace Kratos
+{
+
+Matrix InterfaceStressState::CalculateBMatrix(const Matrix& rDN_DX, const Vector& rN, const Geometry<Node>& rGeometry) const
+{
+    if (rN.empty()) return {};
+
+    Matrix result = ZeroMatrix(GetVoigtSize(), rGeometry.WorkingSpaceDimension() * rGeometry.size());
+
+    for (unsigned int i = 0; i < rGeometry.size() / 2; ++i) {
+        const size_t number_of_u_dofs_per_side = result.size2() / 2;
+
+        result(0, i * rGeometry.WorkingSpaceDimension() + 1)                             = -rN[i];
+        result(0, i * rGeometry.WorkingSpaceDimension() + 1 + number_of_u_dofs_per_side) = rN[i];
+
+        result(1, i * rGeometry.WorkingSpaceDimension())                             = -rN[i];
+        result(1, i * rGeometry.WorkingSpaceDimension() + number_of_u_dofs_per_side) = rN[i];
+    }
+
+    return result;
+}
+
+double InterfaceStressState::CalculateIntegrationCoefficient(const Geometry<Node>::IntegrationPointType& rIntegrationPoint,
+                                                             double                DetJ,
+                                                             const Geometry<Node>& rGeometry) const
+{
+    return rIntegrationPoint.Weight() * DetJ;
+}
+
+Vector InterfaceStressState::CalculateGreenLagrangeStrain(const Matrix& rDeformationGradient) const
```

Comment:
Since this is an unused function parameter, I would suggest to "unname" it:
```suggestion
Vector InterfaceStressState::CalculateGreenLagrangeStrain(const Matrix&) const
```